helm/reana/templates/reana-workflow-controller.yaml

```diff
@@ -68,10 +69,19 @@ spec:
             value: {{ .Values.quota.default_cpu_limit | default 0 | quote }}
           - name: REANA_DEFAULT_QUOTA_DISK_LIMIT
             value: {{ .Values.quota.default_disk_limit | default 0 | quote }}
-          {{- if .Values.workspaces }}{{ if .Values.workspaces.paths }}
+          {{- if not (eq (index .Values.workspaces.paths 0) "/var/reana:/var/reana")}}
+          {{- $default_paths := index .Values.workspaces.paths 0 }}
+          {{- $default_path := split ":" $default_paths }}
           - name: DEFAULT_WORKSPACE_PATH
-            value: {{ index .Values.workspaces.paths 0 }}
-          {{- end }}{{ end }}
+            value: {{ $default_path._1 }}
+          {{- else if ( gt (len .Values.workspaces.paths) 1)}}
+          {{- $default_paths := index .Values.workspaces.paths 1 }}
+          {{- $default_path := split ":" $default_paths }}
+          - name: DEFAULT_WORKSPACE_PATH
+            value: {{ $default_path._1 }}
+          {{- end }}
+          - name: MOUNTED_PATHS
```

Comment:
Why do we need `MOUNTED_PATHS`? I guess if we could expose single `WORKSPACE_PATHS` list and take care about the default values in `reana-commons` we wouldn't need this extra list?